### DISCUSSION TITLE
Need to share data between python and web code.

### DISCUSSION THREAD

**Discussion by sandeepyadav1478** [apache/superset#16006] [Q&A / Help]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Need any way to share information and alter information between python and html code.

**Describe alternatives you've considered**

**Additional context**

**Comment by nytai** [comment]
You can create an api and call it from the frontend

**Comment by mistercrunch** [comment]
It's really unclear what you are trying to do here. Which context/component from the frontend is trying to access what data from the backend?
